### PR TITLE
Updated code to have a provision to set admin connection which maynot be same as currentuser

### PR DESCRIPTION
**Pull request by thathi**
Updated code to support read from appsettings if the admin connection and current connection having different users.

### CODE REVIEW

**Comment by thathi** [comment]
@mVermaat I believe this failure is nothing to do with my changes

**Comment by mVermaat** [comment]
@thathi Thanks for the pull request, I'll take a look into it. Yes, the deployment pipeline failed because the CE instance was unavailable. I'll get back to you.

File: Vermaat.Crm.Specflow/HelperMethods.cs [hunk 1]
```diff
@@ -13,7 +13,10 @@ namespace Vermaat.Crm.Specflow
 {
     public static class HelperMethods
     {
-
+        public static string TryGetAppSettingsValue(string key)
```
Comment (mVermaat) [inline Vermaat.Crm.Specflow/HelperMethods.cs]: I don't really see the need of this method. You might aswell just call GetAppSettingsValue(key, true) directly

Comment (thathi) [reply Vermaat.Crm.Specflow/HelperMethods.cs]: happy to change.  true and false values sometimes confusing have to go to definition to see what that means.

**Pull request merged**